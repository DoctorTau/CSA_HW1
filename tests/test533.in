545
8217 -8579 3751 -6829 969 -2972 -1957 9811 -2487 -9005 7423 7567 612 -1096 3541 -1724 -3179 8921 -2152 -5963 -8566 2741 8171 5483 -5106 -9834 8882 8548 2109 3987 -9307 -1758 -6311 2593 -555 5367 -9636 206 -9971 3605 4174 -5514 -6990 324 -336 -7558 8421 -4098 1147 761 528 2063 5349 885 -6773 -2261 364 481 9099 3125 6060 -9960 7067 9368 485 5650 -8049 -9829 -5226 3070 -7643 2362 -3093 3573 7716 -2424 -2560 -6819 5396 -672 -2707 -9922 -8709 -4561 8305 9177 -5221 51 -8405 -5402 -7736 -9645 -5016 4005 8282 -3286 -5982 4200 8144 -1913 -939 -5295 3948 -5405 -3507 4218 -6822 -9180 416 9624 6788 7240 -9049 -7646 -1372 7816 -6043 -8443 3603 -684 4812 6805 -7559 916 -4830 4345 -2306 -8852 -6328 4890 7569 -1355 -2479 -9991 -4412 -532 -9019 -2453 3661 4755 945 6107 -9088 1661 -382 4813 -7016 3938 -7837 -9751 1643 -176 -7678 -2655 -1155 -4059 8497 202 -4085 -2823 -8303 6907 -9694 -7995 -3466 8654 7323 1014 757 2629 7477 -5385 -4598 2423 4290 -9670 -3758 -3936 -6423 -8078 8389 1631 -4691 -3519 -9096 8024 5703 940 7087 -8640 -2431 -8190 1228 -7708 -4335 5335 -2312 -1414 195 4469 -312 -8401 4581 -870 -4612 4659 5885 -4727 8446 -5598 -5162 -9122 9196 2682 -1869 9819 -2802 3177 -6210 4296 6310 716 6611 -7511 7149 -3315 4580 9663 450 7429 5536 5204 2377 9422 2615 2151 4761 3183 3259 2626 4906 -4723 -2153 2996 -1963 9975 7766 3643 2946 -8078 -101 -6113 -5436 3448 5573 8838 9377 -8790 -9697 -2445 8008 -1282 -7001 5758 2926 4144 5665 656 2972 6572 5544 8201 1387 2723 -8634 4613 -4199 405 4564 815 8381 4609 -8183 -1698 -3940 1452 780 -5919 -531 -4884 -620 -885 3954 4122 -5846 3274 7171 -2654 8332 -7514 -2725 7971 -7942 -3844 -8690 7633 5428 -9502 -750 4244 4211 7561 -2907 -7321 3587 -6165 -4288 9567 6543 -3370 -2669 -8733 8544 6666 7836 213 -3870 4729 -8515 -6737 -6976 9132 2276 983 -7483 7610 -4454 -6157 -2504 -6006 -7743 -7840 -9868 7983 3988 -9878 6100 8293 -1141 3261 -7107 -6814 1946 897 1885 -6887 -5039 4020 8070 -1500 -1590 1302 3577 5536 8720 9555 453 -5476 1130 -1255 7054 -4902 3901 3047 -8686 8560 -8490 5685 -2756 2598 6675 9837 3421 -3355 7921 6994 4551 5224 -2746 1436 7931 -7309 1638 -3928 1104 -6660 452 -9996 2470 6797 5371 -1630 9614 -9302 1922 -2170 6676 532 -6150 -5586 7556 -2382 -6193 -4377 -5360 -161 5648 -4646 -4053 -1474 6630 3930 9615 -7906 585 3553 452 -4590 8355 777 9105 -8228 -9352 -1987 -9847 -3015 8558 -8210 -1875 9975 -1565 2738 1469 2346 7615 -4710 1606 9302 9544 4062 -4036 -7183 -7867 9208 6763 8260 -584 2040 -1243 56 -1094 7667 -6179 -2804 -6710 -7204 -6246 -7265 -9629 -6649 -5132 9674 9464 -7227 -3028 5333 -4038 -9027 -4733 -9025 -903 -936 -5324 -7883 -9958 -1985 -7737 5176 6619 8052 -8920 8553 5859 -6259 -2138 233 -1981 9719 -7529 -5360 -3733 -7916 -3959 9121 -665 -4727 7757 -7720 9935 -6033 1040 2487 1313 1921 155 2960 -1618 -221 6136 -8071 -2676 4055 3657 -2826 -9868 6201 -6154 -5512 9597 -9681 -829 -6421 -2011 7051 -4948 -6172 -3103 -5984 201 2955 -1020 212 1756 6040 4145 