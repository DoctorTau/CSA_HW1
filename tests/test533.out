-2972 -1096 -1724 -2152 -8566 -5106 -9834 -1758 -9636 -5514 -6990 -336 -7558 -4098 -9960 -5226 -2424 -2560 -672 -9922 -5402 -7736 -5016 -3286 -5982 -6822 -9180 -7646 -1372 -684 -4830 -2306 -8852 -6328 -4412 -532 -9088 -382 -7016 -176 -7678 -9694 -3466 -4598 -9670 -3758 -3936 -8078 -9096 -8640 -8190 -7708 -2312 -1414 -312 -870 -4612 -5598 -5162 -9122 -2802 -6210 -8078 -5436 -8790 -1282 -8634 -1698 -3940 -4884 -620 -5846 -2654 -7514 -7942 -3844 -8690 -9502 -750 -4288 -3370 -3870 -6976 -4454 -2504 -6006 -7840 -9868 -9878 -6814 -1500 -1590 -5476 -4902 -8686 -8490 -2756 -2746 -3928 -6660 -9996 -1630 -9302 -2170 -6150 -5586 -2382 -5360 -4646 -1474 -7906 -4590 -8228 -9352 -8210 -4710 -4036 -584 -1094 -2804 -6710 -7204 -6246 -5132 -3028 -4038 -936 -5324 -9958 -8920 -2138 -5360 -7916 -7720 -1618 -2676 -2826 -9868 -6154 -5512 -4948 -6172 -5984 -1020 